dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/RepositoryUtility.java

```diff
@@ -86,6 +87,12 @@
   
   public static final RemoteRepository CENTRAL =
       new RemoteRepository.Builder("central", "default", "http://repo1.maven.org/maven2/").build();
+  public static final RemoteRepository GOOGLE =
```

Comment:
Probably this should be configured at setting.xml of Kokoro user. But I wanted to see how this helps the timeout issue for now.